pkg/cmd/container/inspect.go

```diff
@@ -61,39 +63,6 @@ type containerInspector struct {
 	entries     []interface{}
 }
 
-// resourceTotal will return:
-// - the Usage value of the resource referenced by ID
-// - the cumulative Usage value of the resource, and all parents, recursively
-// Typically, for a running container, this will equal the size of the read-write layer, plus the sum of the size of all layers in the base image
-func resourceTotal(ctx context.Context, snapshotter snapshots.Snapshotter, resourceID string) (snapshots.Usage, snapshots.Usage, error) {
```

Comment:
Promoting this method into `pkg/imgutil` as `ResourceUsage` as it is necessary both for `ps` and `images`.